tests/e2e/pageobjects/ide/Editor.ts

```diff
@@ -481,6 +495,12 @@ export class Editor {
         return By.xpath(`//div[contains(@style, 'top:${lineYCoordinates}px')]//div[contains(@class, 'squiggly-error')]`);
     }
 
+    private async getWarningInLineLocator(lineNumber: number): Promise<By> {
```

Comment:
Does it mean "InLineWarning" ?

Reply:
https://github.com/eclipse/che/pull/19205#discussion_r587369568